### ISSUE TITLE
Mkdocstrings deprecation warnings in dev environment

### ISSUE THREAD

**Issue by cmsirbu** [nautobot/cookiecutter-nautobot-app#113]
Noticed the following deprecation warnings from mkdocstrings in the nautobot-app-dev-example when building docs locally in the dev environment.

```
INFO    -  DeprecationWarning: The `load_module` method was renamed `load`, and is deprecated.
             File
           "/home/vagrant/.cache/pypoetry/virtualenvs/nautobot-dev-example-Uw0SzL5q-py3.11/lib/python3.11/site-packages/mkdocstrings_handlers/python/handler.py",
           line 280, in collect
               loader.load_module(module_name)
             File
           "/home/vagrant/.cache/pypoetry/virtualenvs/nautobot-dev-example-Uw0SzL5q-py3.11/lib/python3.11/site-packages/griffe/loader.py",
           line 105, in load_module
               warnings.warn(
```

**Comment by gsnider2195** [comment]
May have already been resolved by pinning griffe in v2.3.0

**Comment by cmsirbu** [comment]
Confirmed with `develop` in app-dev-example that it doesn't appear anymore. Docs build successfully with code reference pages.
